### PR TITLE
Following Alma changes for stokes$spia, stokes$spr, firstone$pb

### LINKED ISSUE #1650: Stokes location 'spia' and 'spr' have the same external name
Linda Ch. reported in the catalog channel: `Stokes Library houses two collections.  SPIA and SPR.  The catalog no longer shows which collection a book is located in.  Can this please be addressed so we know which section to go to when getting a book`
The external name in Alma for these 2 locations (stokes$spia and stokes$spr) is configured to be the same: 'Wallace Hall'. Can this be updated so that the label indicates which section to go when getting a book? 
@mzelesky @regineheberlein 

When/If the external name in Alma is updated, please open a follow up issue in bibdata to update the locations.

Follow up with Linda Chamberlin.

### PR DESCRIPTION
**Pull request by christinach**
closes #1650 

Update stokes$spia, stokes$spr labels following Alma changes
Update firestone$pb location label change following Alma changes

### CODE REVIEW

File: marc_to_solr/translation_maps/locations.rb.tmpl [hunk 1]
```diff
@@ -80,7 +80,8 @@
  "firestone$prne"=>"Near East Periodicals Collection",
  "firestone$pr"=>"Periodicals Collection",
  "firestone$raas"=>"African American Studies Reading Room: Reserve",
- "firestone$pb"=>"Remote Storage: Supervised use in Firestone A-16-H-3",
+ "firestone$pb"=>
+  "Remote Storage (ReCAP): Supervised use in Firestone A-16-H-3",
```
Comment (hectorcorrea) [inline marc_to_solr/translation_maps/locations.rb.tmpl:84]: Is the extra line break OK?

Comment (escowles) [inline marc_to_solr/translation_maps/locations.rb.tmpl:84]: Let me see if I can get some input on what we should say instead of the non-existent A-16-H-3

**Review by escowles — CHANGES_REQUESTED** [review]

File: marc_to_solr/translation_maps/locations.rb.tmpl (hunk 1 above)
Comment (christinach) [reply marc_to_solr/translation_maps/locations.rb.tmpl:84]: @escowles The change of the non existent 'A-16-H-3' needs to happen in Alma. I dont want to block this pr for this part. The pr is mainly for stokes library.

Comment (christinach) [reply marc_to_solr/translation_maps/locations.rb.tmpl:84]: @hectorcorrea yes it is ok.

**Review by hectorcorrea — APPROVED** [review]

**Review by escowles — APPROVED** [review]

**Pull request merged**